test/e2e/runner/start.go

```diff
@@ -102,15 +119,16 @@ func Start(ctx context.Context, testnet *e2e.Testnet) error {
 
 		logger.Info("Starting catch up node", "node", node.Name, "height", node.StartAt)
 
-		if err := execCompose(testnet.Dir, "up", "-d", node.Name); err != nil {
+		err := p.StartNodes(context.Background(), node)
```

Comment:
in the previous loop, logically, we should first `waitForHeight(... node.StartAt ...)` and then update `networkHeight`.

Reply:
Do you mean the loop starting at line 54? Those nodes start all at height zero.
I had to change the logic a bit here, to make the DO way of doing things compatible with Docker:
* Before: nodes that start at height zero were started one by one. That is very inefficient with a large testnet (think of the 200-node testnet)
* After: nodes starting at height zero are all started at the same time. Then we wait for all of them to be started (`waitForNode` with height zero)